web/ts/TUMLiveVjs.ts

```diff
@@ -481,6 +488,21 @@ function debounce(func, timeout) {
     };
 }
 
+// iPhone/iPad need to set the progress again when they actually play the video. That's why loadedmetadata is
+// not sufficient here.
+// See https://stackoverflow.com/questions/28823567/how-to-set-currenttime-in-video-js-in-safari-for-ios.
+// Returns updated iOSReady value
+function setTimeOnIOS(iOSReady: boolean, seconds: number): boolean {
+    player.on("canplaythrough", () => {
+        // Can be executed multiple times during playback
+        if (!iOSReady) {
+            player.currentTime(seconds);
+            return true;
```

Comment:
In relation to #752 : this doesn't look right. From what I can tell, the `return true;` is meant to be used to update the value of `iOSReady` in `watchProgress`. Returning it from the callback doesn't do that. Thus, `iOSReady` is always false and the player keeps settings the current time (in the case of #752 , to the end of the video). Can't test if that's actually the problem right now.